apsuite/commisslib/meas_touschek_lifetime.py

```diff
@@ -70,23 +78,55 @@ def __str__(self):
 
 
 class MeasTouschekLifetime(_BaseClass):
-    """."""
+    """Measurement of two single-bunches current decay with BPM sum signal.
+
+    In single-bunch mode, RFFE attenuation must be adjusted in all BPMs
+    before injection, the default value is 30dB. The BPM used in the
+    measurement is allowed to have an attenuation of 14dB to increase the sum
+    signal, if the maximum single-bunch current is 3.0mA (to be within the
+    recommendation of 10% of sum signal full scale which is 2^15 counts). In
+    the case that some BPMs attenuation setting fails, a local reset of RFFE
+    module at the BPM rack must be done.
+
+    The measurement is typically performed with two single-bunches with
+    different currents: one with high current (~2mA) and one with low current
+    (~0.2mA).
+
+    It was observed that the calibration curves BPM Sum -> Current and also the
+    DCCT offset considerably changed during a time span of about 5 months (
+    observed between machine studies in November 2021 and April 2022).
+    Therefore, it is recommended to re-measure the calibration curves prior to
+    the start of the experiment.
+    [TO-DO]: Bring the calibration curve measurement and analysis from jupyter-
```

Comment:
@murilo, you can use comments starting with  `# NOTE:` in order to signal to-do activities. Code understands with signalling. maybe remove this part from docstring and add as a NOTE comment...

Reply:
done, thanks! I was wondering how to include these to-do activities. I didn't know the TODO signalling in python, thank you!